src/Heading/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
This import statement is not needed here. So you can remove it.

Reply:
removed